gnomad_qc/v4/annotations/insilico_predictors.py

```diff
@@ -508,6 +509,90 @@ def create_phylop_grch38_ht() -> hl.Table:
     return ht
 
 
+def get_revel_for_unmatched_transcripts() -> None:
+    """Create Tables with REVEL scores for variants in v4.1 release in Ensembl v105 MANE Select transcripts that are not found in REVEL (Ensembl v64)."""
```

Comment:
Can you expand on this some and explain how the behavior differs (getting max at locus and alleles vs transcript) in the docstring?